server/src/main/java/org/opensearch/OpenSearchException.java

```diff
@@ -1618,6 +1618,12 @@ private enum OpenSearchExceptionHandle {
             org.opensearch.cluster.decommission.NodeDecommissionedException::new,
             164,
             V_3_0_0
+        ),
+        INDEX_CREATE_BLOCK_EXCEPTION(
+            org.opensearch.cluster.block.IndexCreateBlockException.class,
+            org.opensearch.cluster.block.IndexCreateBlockException::new,
+            165,
+            UNKNOWN_VERSION_ADDED
```

Comment:
Any specific reason for UNKNOWN Version?

Reply:
I have not associated this exception with any ES version similar to other blocks: https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/OpenSearchException.java#L1045